localisation/touhou_events_l_english.yml

```
@@ -5515,6 +5515,9 @@
  flavor_jnk.1008.t: ""
  flavor_jnk.1008.desc: ""
  flavor_jnk.1008.a: "" 
+ flavor_jnk.1009.t: "Grudge toward traitors"
+ flavor_jnk.1009.desc: "As Junko's grudge grew up, tension and fear began to creep into our court. Fortunately, her wise servants devised a way to vent some of her grudge while she still had a shred of reason left. They went to the furious queen and reported that there were those in every corner of the country who were disturb her vengeance, and implored her to pour out her anger on those foolish traitors. She storm out of the court before even they could finish speaking, and it wasn't long before word came that the rebellions that had broken out in various parts of our country were beginning to be put down..."
```

Some suggestion for new line characters, also changing "grudge grew up" to "grudge grew"
```suggestion
 flavor_jnk.1009.desc: "As Junko's grudge grew, tension and fear began to creep into our court. Fortunately, her wise servants devised a way to vent some of her grudge while she still had a shred of reason left.\n\nThey went to the furious queen and reported that there were those in every corner of the country who were disturb her vengeance, and implored her to pour out her anger on those foolish traitors. \nShe storm out of the court before even they could finish speaking, and it wasn't long before word came that the rebellions that had broken out in various parts of our country were beginning to be put down..."
```

---

accept most of feedback and added description for government mechanism briefing event.